packages/@aws-cdk/aws-ec2/rosetta/with-vpc.ts-fixture

```diff
@@ -0,0 +1,13 @@
+// Fixture with packages imported and a VPC created
```

Comment:
wouldn't it make more sense to use `.fixture.ts` extension so the IDE/github will know this is typescript?

Reply:
Maybe, but the problem is the include paths won't resolve in the location where the files are stored so the IDE will start complaining and showing permanent errors in the `Errors` tab. 

I've decided against that for now.